Dockerfile

```diff
@@ -14,11 +14,12 @@ ENV NODE_OPTIONS=--max_old_space_size=8000
 
 WORKDIR /tmp/grafana
 
-COPY package.json yarn.lock .yarnrc.yml ./
+COPY package.json project.json nx.json yarn.lock .yarnrc.yml ./
 COPY .yarn .yarn
 COPY packages packages
 COPY plugins-bundled plugins-bundled
 COPY public public
+COPY LICENSE ./
```

Comment:
why are we moving this? if we need it here then we'll also need to update line 182 to copy it from js-src instead of go-src

Reply:
Thanks for spotting this. The decoupled plugins webpack config copies the License into their dist directories. Without it webpack will error out and builds fail. I've gone with your suggestion to update it to `--from=js-src` which is working for me locally.